yateto/codegen/copyscaleadd/csa_gen.py

```diff
@@ -121,7 +122,8 @@ def __eq__(self, other):
       return False
 
   def header(self, cpp):
-    cpp.include('gemmforge_aux.h')
+    for header in self._headers:
+      cpp.include(header)
```

Comment:
This is fine but this pattern happens more often. Maybe it would be easier to add a method
`cpp.includes(self._headers)`? Just an idea - not sure whether this pattern is used more often than in this PR

Reply:
done